Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -146,6 +146,36 @@ struct SymbolGraphLoader {
         
         self.symbolGraphs = loadedGraphs.mapValues(\.graph)
         (self.unifiedGraphs, self.graphLocations) = graphLoader.finishLoading()
+        
+        unifiedGraphs.forEach { (_, unifiedGraph) in
+            var unifiedGraph = unifiedGraph
+            var defaultUnconditionallyUnavailablePlatforms = [PlatformName]()
+            var defaultAvailableAvailabilities = [DefaultAvailability.ModuleAvailability]()
+
+            if let defaultAvailabilities = bundle.info.defaultAvailability?.modules[unifiedGraph.moduleName] {
+                // Availability platforms declared in the Info.plist as `unavailable`.
+                defaultUnconditionallyUnavailablePlatforms = defaultAvailabilities.filter {
+                    $0.state == .unavailable
+                }.map(\.platformName)
+                // Availability platforms declared in the Info.plist as `available`.
+                defaultAvailableAvailabilities = defaultAvailabilities.filter {
+                    $0.state != .unavailable
+                }
```

Comment:
minor: We have a nice `categorize(where:)` extension that simplifies cases like this where one value represent all the `true` cases and the other all the `false` cases 
```suggestion
                let (unavailablePlatforms, availablePlatforms) = defaultAvailabilities.categorize(where: { $0.state == .unavailable })
                
                platformsToMarkUnavailableAsFallback = unavailablePlatforms.map(\.platformName)
                defaultIntroducedVersions = availablePlatforms
```

Reply:
This makes it more concise and easy to read, thx for the suggestion! https://github.com/apple/swift-docc/commit/c55abc329895424275b96a3c47afef3f8c5bb3f9